### DISCUSSION TITLE
@types/react-image-gallery is missing useWindowKeyDown prop

### DISCUSSION THREAD

**Discussion by FunctionDJ** [DefinitelyTyped/DefinitelyTyped#54659] [Issues with a @types package]
Props: https://github.com/xiaolin/react-image-gallery#props
File in DT where this prop is missing: https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react-image-gallery/index.d.ts

**Comment by danactive** [comment]
This is the workaround I used
```
declare module 'react-image-gallery' {
  interface ReactImageGalleryProps {
    useWindowKeyDown?: boolean;
  }
}
```
